src/features/Mission/missionSlice.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-param-reassign */
+// Import createSlice() from Redux toolkit:
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from '../../axios';
+
+// Initial state for Redux store:
+const initialState = {
+  missions: [],
+  isLoading: true,
+  isMember: true,
+  status: 'idle',
+  error: '',
+};
+
+export const fetchMissions = createAsyncThunk(
+  'missions/fetchMissions',
+  async () => {
+    const response = await axios.get('/missions');
+    const missionsArray = Object.keys(response.data).map((key) => ({
+      mission_id: key,
```

Comment:
- [ ] [OPTIONAL] It would be better if you can use the duck pattern in your folder structure, you can use something like the following:
```
src
|   ...
|   redux
|   |   componentSlice1
|   |   |   ...
|   |   componentSlice2
|   |   |   ...
|   |   configureStore
```